go-apps/meepctl/cmd/deploy.go

```diff
@@ -193,6 +201,15 @@ func deployCore(cobraCmd *cobra.Command) {
 			coreFlags = utils.HelmFlags(coreFlags, "--set", "user.swagger.enabled=true")
 			coreFlags = utils.HelmFlags(coreFlags, "--set", "user.swagger.location="+deployData.workdir+"/user/swagger")
 		}
+		if userValues {
+			// user provided overriding values
+			// Note: according to https://helm.sh/docs/chart_template_guide/values_files/
+			//       the order of precedence is: (lowest) default values.yaml
+			//                                            then user value file
+			//                                            then individual --set params (highest)
+			//       Therefore, the --set flags inserted by meepctl may interfere with user overrides
+			coreFlags = utils.HelmFlags(coreFlags, "-f", userValueFile)
+		}
```

Comment:
The user values file is added in the function deployRunScriptsAndGetFlags(). Adding the flag here should not be necessary.

Reply:
Right... moved it but forgot to clean this one.
Will update